modules/nodes-scheduler-profiles-configuring.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// * nodes/scheduling/nodes-scheduler-profiles.adoc
+
+[id="nodes-scheduler-profiles-configuring_{context}"]
+= Configuring a scheduler profile
+
+You can configure the scheduler to use a scheduler profile.
+
+[NOTE]
+====
+Do not set both a scheduler policy and a scheduler profile. If both are set, the scheduler policy takes precedence.
+====
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Edit the `Scheduler` object:
++
+[source,terminal]
+----
+$ oc edit scheduler cluster
+----
+
+. Specify the profile to use in the `spec.profile` field:
++
+[source,yaml]
+----
+apiVersion: config.openshift.io/v1
+kind: Scheduler
+metadata:
+  ...
+  name: cluster
+  resourceVersion: "601"
+  selfLink: /apis/config.openshift.io/v1/schedulers/cluster
+  uid: b351d6d0-d06f-4a99-a26b-87af62e79f59
+spec:
+  mastersSchedulable: false
+  policy:
+    name: ""
+  profile: LowNodeUtilization <1>
+----
+<1> Set to `LowNodeUtilization`, ``HighNodeUtilization`, or `NoScoring`.
++
+// TODO: Specify that LowNodeUtilization is the default? Is it actually the default, or should we say to use it for the default scheduler behavior?
```

Comment:
The default profile is, technically, no profile (which just uses the kube-scheduler's builtin default registry of enabled plugins). That default registry effectively equates to a `LowNodeUtilization` profile, where pods are spread out to minimize node usage.

I'm not sure how the best way to word that for docs is though. Maybe just make it clear that LowNodeUtilization is equivalent to the scheduler's default off-the-shelf behavior?

Reply:
@damemi Thanks, I updated the other section where I list the profiles to say "This profile provides the default scheduler behavior." Let me know if that sounds okay to you.

https://deploy-preview-28893--osdocs.netlify.app/openshift-enterprise/latest/nodes/scheduling/nodes-scheduler-profiles.html#nodes-scheduler-profiles-about_nodes-scheduler-profiles

Reply:
That sounds great, thanks